src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -87,87 +86,80 @@
     private static final double HALF_BLOCK_OFFSET = 0.5D;
 
     /**
-     * The time in ticks the lumberjack has waited already.
-     * Directly connected with the MAX_WAITING_TIME.
+     * Number of ticks to wait for tree.
      */
-    private int timeWaited = 0;
-
+    private static final int TIMEOUT_DELAY     = 10;
+    private static final int LEAVES_RADIUS     = 3;
+    private static final int ITEM_PICKUP_RANGE = 3;
+    private static final int STUCK_WAIT_TICKS  = 20;
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)